Microsoft.DotNet.Interactive.Jupyter.Tests/ExecuteRequestHandlerTests.cs

```diff
@@ -324,6 +325,27 @@ public async Task sends_InputRequest_message_when_submission_requests_user_input
                 .Contain(dp => dp.Data["text/html"] as string == expectedDisplayValue);
         }
 
+        [Theory]
+        [InlineData("Read-Host", "", "input-value")]
+        [InlineData("Read-Host -Prompt User", "User: ", "user name")]
+        public async Task sends_InputRequest_message_when_submission_requests_user_input_in_powershell(string code, string prompt, string expectedDisplayValue)
```

Comment:
In `LanguageKernelTests` and elsewhere, we've combined tests for different languages into a single `Theory` test in order to check that the experience aligns across languages. This approach can work if the assertions are the same regardless of language. 

Comparing this test to the C# one above, I wonder about the differences in the assertion. For example, does the mime type get set the same way for PowerShell?

Reply:
PowerShell has its own formatting system that generates texts representing the result objects, which are then rendered by the host. The captured stdout is exactly what the PowerShell host needs, so it can directly use `Console.Write` and `Console.WriteLine` to write out the texts. So, the resulted `PubSubMessage` is `Stream` instead of `ExecuteResult`.